packages/popover/src/Popover.story.tsx

```diff
@@ -123,6 +123,65 @@ function AdvancedExample() {
   );
 }
 
+function ScrollExample() {
+  const [portalContainer, setPortalContainer] = useState<HTMLDivElement | null>(
+    null,
+  );
+  const [active, setActive] = useState(false);
+  const buttonRef = useRef(null);
+
+  return (
+    <>
+      <div
+        ref={el => setPortalContainer(el)}
+        className={css`
+          margin: 150px;
+          width: 100%;
+          overflow-y: auto;
```

Comment:
not urgent but this could be a className

Reply:
For css declarations in storybook that aren't shared across pages, I don't mind inline TBH. Let me know if you feel strongly about this though and I can update it.

Reply:
that's okay